src/ExpandableDataPicker/ExpandableDataPicker.js

```diff
@@ -70,6 +70,7 @@ module.exports = kind(
 	*/
 	collectionChanged: function () {
 		this.$.list.set('collection', this.collection);
+		if (this.collection && !this.$.list.hasNode()) this.$.list.refresh();
```

Comment:
```this.$.list.refresh()``` will render each items in collection. I think an advantage of ExpandableDataPicker comparing to ExpandablePicker is to not create/render child controls before picker is expanded

Is there another fix not creating every child controls of picker?